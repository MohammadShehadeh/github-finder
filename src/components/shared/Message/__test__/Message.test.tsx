import renderer from 'react-test-renderer';

import { Message } from '..';

const mockMessageProps = {
  title: 'title',
  description: 'description',
};

describe('Message', () => {
  it('should render to match snapshot', () => {
    const tree = renderer.create(<Message {...mockMessageProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
