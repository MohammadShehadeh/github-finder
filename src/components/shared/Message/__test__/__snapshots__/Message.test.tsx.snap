// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should render to match snapshot 1`] = `
<div
  className="message"
>
  <h2
    className="title"
  >
    title
  </h2>
  <p
    className="description"
  >
    description
  </p>
</div>
`;
